README.md

```diff
@@ -1,8 +1,34 @@
 
 # penalties-file-notification-orchestrator
 
-This is a placeholder README.md for a new repository
+PENALTIES FILE NOTIFICATION ORCHESTRATOR has 2 endpoints
 
-### License
+POST        /new-notifications                    controllers.OrchestratorController.receiveSDESNotifications
+new-notifications is a new notification service for backend services.
+
+
+POST        /sdes-callback                        controllers.SDESCallbackController.handleCallback
+sde will pull from file upload, process the file and message is sent back with the status.
+
+
+## Running
+
+This application runs on port 9184.
+
+You can use the ./run.sh to run the service.
+
+The user must have an authenticated session and be enrolled in MTD VAT to access most pages of this service.
```

Comment:
There is no authentication mechanism in this service so this can be removed
```suggestion
```
```suggestion
The user must have an authenticated session and be enrolled in MTD VAT to access most pages of this service.
```

Reply:
Could you remove this please?